psalm.xml.dist

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0"?>
+<psalm
+    errorLevel="8"
+    resolveFromConfigFile="true"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns="https://getpsalm.org/schema/config"
+    xsi:schemaLocation="https://getpsalm.org/schema/config vendor/vimeo/psalm/config.xsd"
+>
+    <projectFiles>
+        <ignoreFiles>
+            <directory name="vendor"/>
+        </ignoreFiles>
+        <directory name="CacheWarmer"/>
+        <directory name="Command"/>
+        <directory name="Controller"/>
+        <directory name="DataCollector"/>
+        <directory name="Dbal"/>
+        <directory name="DependencyInjection"/>
+        <directory name="EventSubscriber"/>
+        <directory name="Mapping"/>
+        <directory name="Repository"/>
+        <directory name="Tests"/>
+        <directory name="Twig"/>
+        <file name="ConnectionFactory.php"/>
+        <file name="DoctrineBundle.php"/>
+        <file name="ManagerConfigurator.php"/>
+        <file name="Registry.php"/>
```

Comment:
Lost of directories in files ignored, but I suppose that's temporary, so that we can start having Psalm and remove them one by one?

Reply:
These are inclusions, not exclusions actually. DoctrineBundle doesn't have central directory like `src` or `lib` at the moment.

Reply:
Oh right, my bad!